packages/cspell-lib/package.json

```diff
@@ -48,6 +48,8 @@
   },
   "homepage": "https://github.com/streetsidesoftware/cspell#readme",
   "dependencies": {
+    "@types/fs-extra": "^9.0.3",
```

Comment:
Just like above. Let's avoid having types in the `dependencies` section. If they are needed, let's find out why.

Reply:
`Stats` from 'fs-extras' is exported here https://github.com/streetsidesoftware/cspell/blob/c8b115db4916f4d31ebfc10972437030e71fac3f/packages/cspell-lib/src/SpellingDictionary/DictionaryLoader.ts#L49